site/content/how-to/traffic-management/snippets-filters.md

```diff
@@ -0,0 +1,345 @@
+---
+title: "Snippets Filters"
+weight: 800
+toc: true
+docs: "DOCS-000"
+---
+
+Learn how to use Snippets with the `SnippetsFilter` API.
+
+## Overview
+
+Snippets allow Cluster Operators to insert NGINX configuration into different contexts of the
```

Comment:
```suggestion
Snippets allow users to insert NGINX configuration into different contexts of the
```

Cluster operators create them, application developers use them. Probably best to keep it generic here.